File: 04/HF/PyUtils.java
import java.util.ArrayList;
import java.util.List;

public class PyUtils {
    private PyUtils() {
        // nem példányosítható
    }

    public static List<Integer> range(int hi) {
        return range(0, hi, 1);
    }

    public static List<Integer> range(int lo, int hi) {
        return range(lo, hi, 1);
    }
    
    public static List<Integer> range(int lo, int hi, int step){
        List<Integer> result = new ArrayList<>();

        for(int i = lo; i < hi; i += step){
            result.add(i);
        }

        return result;
    }
    
}

File: 14/HF/ListComp/ListComp/PyUtils.cs
using System.Collections.Generic;

namespace ListComp
{
    public class PyUtils
    {
        private PyUtils(){
            
        }

        public static List<int> Range(int hi)
        {
            return Range(0, hi, 1);
        }

        public static List<int> Range(int lo, int hi)
        {
            return Range(lo, hi, 1);
        }

        public static List<int> Range(int lo, int hi, int step)
        {
            List<int> list = new List<int>();

            for (int i = lo; i < hi; i += step)
            {
                list.Add(i);
            }

            return list;
        }
    }
}